katas/content/key_distribution/random_array/Verification.qs

```diff
@@ -6,8 +6,8 @@ namespace Kata.Verification {
     operation CheckSolution() : Bool {
         // The test only checks that the operation returns an array of correct length
         // and that it's not always the same. It doesn't analyze the distribution of true and false elements.
-        let N = 10;
-        let randomArrays = ForEach(Kata.RandomArray, [N, N, N]);
+        let N = 20;
+        let randomArrays = ForEach(Kata.RandomArray, [N, size = 10]);
```

Comment:
Did you mean to change N from 10 to 20, but then pass `size = 10`.

Reply:
N is length of individual array, size is the number of arrays we sample. They don't have to be the same.